test/CloudNative.CloudEvents.UnitTests/Kafka/KafkaTest.cs

```diff
@@ -156,17 +208,16 @@ public override bool CanConvert(Type objectType)
                 {
                     return null;
                 }
-                else 
+
```

Comment:
Is there any actual change to the code here? It seems unrelated to the PR.

Reply:
Purely structure. Should I revert to if / else instead of guard?